swagger/vra-blueprint.json

```diff
@@ -5,7 +5,7 @@
         "version": "2019-09-12",
         "title": "VMware Cloud Assembly Blueprint API"
     },
-    "host": "api.mgmt.cloud.vmware.com",
+    "host": "cava-r-89-003.eng.vmware.com",
```

Comment:
I do not see this in iaas json file. If this instance is deleted, it should not cause any problem, right?

Reply:
It's being used as the default endpoint if the client does not specify it. I'm assuming each client specifies their own vRA instance endpoint, otherwise, they won't be able to reach this cava endpoint. Anyway, I believe it makes more sense to default to the vRA Cloud instance. I'll update this.

BTW, it is already at the iaas swagger file: https://github.com/vmware/vra-sdk-go/blob/master/swagger/vra-iaas.json#L10768. . I'll fix it too.